packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.js

```diff
@@ -121,6 +122,8 @@ const GiftCards = props => {
                         field="card"
                         message={cardEntryErrorMessage}
                         placeholder={'Enter card number'}
+                        validate={isRequired}
+                        validateOnBlur
```

Comment:
I would not validate on blur since simply tabbing through the gift card section does not imply that you actually want to add a card.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2175/commits/b1fbf46e455a48b1dff9c6b0759c8ddbe0cf7a9a.